### PR TITLE
Show react hooks usage

### PR DESCRIPTION
**Pull request by nadilas**

### CODE REVIEW

File: src/components/AppLayout.tsx [hunk 1]
```diff
@@ -39,14 +43,14 @@ export default function AppLayout({ children }: React.PropsWithChildren) {
         <Disclosure as="nav" className="bg-gray-800">
           {({ open }) => (
             <>
-              <div className="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
-                <div className="flex h-16 items-center justify-between">
+              <div className="px-4 mx-auto max-w-7xl sm:px-6 lg:px-8">
+                <div className="flex items-center justify-between h-16">
```
Comment (tasn) [inline src/components/AppLayout.tsx]: There are a lot of unrelated changes in this PR that make it really really hard to review. :)

Comment (nadilas) [reply src/components/AppLayout.tsx]: On it

File: src/pages/dashboard/messages.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { hardcodedUsername, postWithAuth } from "@/auth";
```
Comment (tasn) [inline src/pages/dashboard/messages.tsx]: Can you add documentation both here, the component, and the README that this is a svix-react example?

To be honest, I'd probably not call the files Messages but rather SvixReactExample, or maybe put it in a subdirectory so it doesn't confuse people who don't want it / need it. Because it's more advanced.

Actually, maybe put it in an "advanced" directory or something?

I just really don't want to scare away people, they need to understand how simple Svix is. :)

**Review by tasn — APPROVED** [review]
